### ISSUE TITLE
Linux mint not recognised for dependency installation in install script.

### ISSUE THREAD

**Issue by hamishmb** [gitbls/pistrong#11]
Hi there,

Just a quick note to say that linux mint isn't recognised by your install script:

`% Pre-reqs unknown for distro linuxmint`

These will just be the same as for Ubuntu as mint is based on Ubuntu :)

**Comment by gitbls** [comment]
Hey, thanks for this. Will look into adding (and testing) it. 

In the meantime, were you able to work around this and get the VPN running on your Mint system?

**Comment by hamishmb** [comment]
No problem.

Yes, I just continued anyway and it worked fine.

**Comment by gitbls** [comment]
Did a "quick fix" to enable distros with ID_LIKE=ubuntu in /etc/os-release. A proper fix is awaiting proper support for other distros.

**Comment by hamishmb** [comment]
Excellent, thanks, that should fix it for me. Supporting a lot of distros is hard, and honestly I reckon this is probably enough for almost all of them.

**Comment by hamishmb** [comment]
Shall I close this?

**Comment by gitbls** [comment]
Sure, go ahead and close it.

JFYI I'm still planning to add more distros (it used to, but I removed them when I overhauled InstallPiStrong to support both install from repo and download/build/install from source. Just haven't gotten around to adding the other distros back, and there haven't been any requests. For now, distros with ID_LIKE=ubuntu should all work without getting the "I don't recognize your distro" message.
